### PR TITLE
CDAP-2653 fix the twitter schema to correctly account for nullable

### PR DESCRIPTION
**Pull request by albertshau**
fields. Also updating the plugin description to give information
about the output of the source, indicating field types and whether
or not they are nullable.

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/TwitterSource.java [hunk 1]
```diff
@@ -48,7 +48,9 @@
  */
 @Plugin(type = "source")
 @Name("Twitter")
-@Description("Twitter Realtime Source")
+@Description("Twitter Realtime Source. Output records will always contain id (long), message (string), " +
+  "favCount (int), rtCount (int), and isRetweet (boolean) fields. Output records may contain lang (string), " +
+  "time (long), source (string), geolat (double), and geoLong (double) fields if they are available.")
```
Comment (gokulavasan) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/TwitterSource.java]: Not part of this PR but can we rename the GLAT field from geolat to geoLat ? to be in sync with geoLong.

**Comment by gokulavasan** [comment]
One comment. Fix and merge. Thanks! LGTM :+1:

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/TwitterSource.java [hunk 2]
```diff
@@ -119,15 +121,15 @@ private StructuredRecord convertTweet(Status tweet) {
...
-    } else {
-      recordBuilder.set(GLAT, -1d);
-      recordBuilder.set(GLNG, -1d);
     }
     recordBuilder.set(ISRT, tweet.isRetweet());
     return recordBuilder.build();
   }
 
-  private long convertDataToTimeStamp(Date date) {
+  private Long convertDataToTimeStamp(Date date) {
```
Comment (bdmogal) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/TwitterSource.java]: mark this ``@Nullable``?
